app/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -242,5 +242,25 @@ public static int getIntOrDefault(String value, int defaultValue) {
             throw new BadRequestException(value + " is not an integer");
         }
     }
-    
+
+    /**
+     * Creates a new copy of the map, removing any entries that have a null value (particularly easy to do this in
+     * JSON).
+     */
+    public static <K,V> Map<K,V> withoutNullEntries(Map<K,V> map) {
```

Comment:
Probably, but I don't think it'll help in this specific case. We want to remove null entries from the map from the data itself, not just when it's being serialized to JSON.